src/main/java/com/github/fabienrenaud/jjb/stream/Serialization.java

```diff
@@ -93,8 +95,11 @@ public Object genson() throws Exception {
 
     @Benchmark
     @Override
-    public Object jsonio() throws Exception {
-        return com.cedarsoftware.util.io.JsonWriter.objectToJson(JSON_SOURCE().nextPojo(), JSON_SOURCE().provider().jsonioStreamOptions());
+    public Object jsonio() {
+        /**
+         * showTypeInfoNever maps to old TYPE=false behavior see {@link JsonIo#getWriteOptionsBuilder(java.util.Map)}
```

Comment:
nit: use single line comment style.

Reply:
Done, amended in 604f78f (losing code navigation in IDE though).